apps/charterafrica/src/components/Tool/DescriptionAndShare.js

```diff
@@ -18,18 +18,20 @@ const DescriptionAndShare = React.forwardRef(
         >
           {description}
         </RichTypography>
-        <RichTypography
-          textAlign="left"
-          color="neutral.dark"
-          sx={{
-            mt: 3.75,
-            textTransform: "none",
-            textAlign: { xs: "center", sm: "left" },
-          }}
-          variant="captionCap"
-        >
-          {activeText} {lastActive}
-        </RichTypography>
+        {lastActive ? (
+          <RichTypography
+            textAlign="left"
+            color="neutral.dark"
+            sx={{
+              mt: 3.75,
+              textTransform: "none",
+              textAlign: { xs: "center", sm: "left" },
+            }}
+            variant="captionCap"
+          >
+            {activeText} {lastActive}
```

Comment:
We can create `lastActiveText` before `return` above
```js
  const lastActiveText = lastActive ? `${activeText} ${lastActive}` : null
```
Then, we don't need `{lastActive ? (` around the `RichTypography` since it can handle `null`/empty:
```suggestion
            {lastActiveText}
```